app/src/main/java/com/github/steroidteam/todolist/view/NoteDisplayActivity.java

```diff
@@ -34,4 +42,31 @@ protected void onCreate(Bundle savedInstanceState) {
                         + " voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non"
                         + " proident, sunt in culpa qui officia deserunt mollit anim id est laborum.");
     }
+
+    public void pickFile(View view) {
+        Intent intent = new Intent();
+        intent.setType("image/*");
+        intent.setAction(Intent.ACTION_GET_CONTENT);
+        startActivityForResult(Intent.createChooser(intent, "Select Picture"), PICK_IMAGE);
+    }
+
+    @Override
+    public void onActivityResult(int requestCode, int resultCode, Intent data) {
+        super.onActivityResult(requestCode, resultCode, data);
+        if (requestCode == PICK_IMAGE) {
+            Uri uri = data.getData();
+            ConstraintLayout header = findViewById(R.id.note_header);
+            Bitmap bitmap = null;
+            try {
+                InputStream is = getContentResolver().openInputStream(uri);
+                bitmap = BitmapFactory.decodeStream(is);
+                is.close();
+            } catch (Exception e) {
+            }
+            RoundedBitmapDrawable ob = RoundedBitmapDrawableFactory.create(getResources(), bitmap);
+            ob.setCornerRadius(70);
```

Comment:
We are using 25dp-radius corners in the rest of the UI, can we keep it here for consistency? I don't know if unit conversion might be needed (not sure about the units of that 70).

Reply:
Indeed it is possible by using `getResources().getDisplayMetrics().density`